### DISCUSSION TITLE
v9: Theme Designer

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#33186] [Q&A]
Hi,

I have been playing with the v9 theme designer for selecting an appropriate color scheme for my application. It seems that the color selected from the color picker is not directly used in any of 16 shades generated by the designer. 

One would expect that the selected color using the color picker is the "primary" brand color and not a shade generated from the selected color.

How can I fix/change this behaviour?   

Thanks.

**Comment by hassanu-t** [comment]
I am having the same issue, I enter my color and it generates a palette without my main color in it!

**Comment by blendsdk** [comment]
I think we need to dig into this function for some answers:

[https://github.com/microsoft/fluentui/blob/70d3d23742e1914983cde0c9a16bb36e1a35165a/packages/react-components/theme-designer/src/utils/getBrandTokensFromPalette.ts#L25](https://github.com/microsoft/fluentui/blob/70d3d23742e1914983cde0c9a16bb36e1a35165a/packages/react-components/theme-designer/src/utils/getBrandTokensFromPalette.ts#L25)
